src/PrestaShopBundle/Resources/config/services.yml

```diff
@@ -71,6 +71,9 @@ services:
     prestashop.adapter.image_manager:
         class: PrestaShop\PrestaShop\Adapter\ImageManager
 
+    prestashop.adapter.data_provider.attribute:
```

Comment:
Please add the service in the right category (# Data providers service, not # Other adapters)